src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Unix.cs

```diff
@@ -293,6 +293,21 @@ public bool UseDefaultCredentials
                 {
                     _curlHandler.UseDefaultCredentials = value;
                 }
+                else
+                {
+                    if (value)
+                    {
+                        _socketsHttpHandler.Credentials = CredentialCache.DefaultCredentials;
+                    }
+                    else
+                    {
+                        if (_socketsHttpHandler.Credentials == CredentialCache.DefaultCredentials)
```

Comment:
Shouldn't we have similar logic for `get` above?

Reply:
Here's the code on Windows.  Can we add similar logic, and a similar comment as well?

```csharp
            // WinHttpHandler doesn't have a separate UseDefaultCredentials property.  There
            // is just a ServerCredentials property.  So, we need to map the behavior.
            // Do the same for SocketsHttpHandler.Credentials.
            //
            // This property only affect .ServerCredentials and not .DefaultProxyCredentials.

            get => _winHttpHandler != null ? _winHttpHandler.ServerCredentials == CredentialCache.DefaultCredentials :
                    _socketsHttpHandler.Credentials == CredentialCache.DefaultCredentials;
```

Reply:
Yes, good catch